### DISCUSSION TITLE
how to reuse a github repository name that has been retired

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68859] [Repositories] [answered]
### Select Topic Area

Question

### Body

The repository github.com/globaldce/globaldce has been retired and cannot be reused. Please, can you delete this retired repo so I can reuse this name?

**Comment by vishalkrishna8** [comment]
Archive or delete the retired repository from your GitHub account. After doing so, wait for some time as it may take a little while for GitHub to fully remove the repository name from their system. Once you're certain the name is no longer associated with the retired repository, you should be able to create a new repository using that name. Hope this helps you @mr9184

**Reply by ghost** [reply]
Thanks for the reply. I already deleted this repo, yet I am still waiting to be able to reuse the name.

**Reply by vishalkrishna8** [reply, marked as answer]
It may take some time for GitHub to release the name for reuse. The exact duration can vary, and it's typically handled on GitHub's end. You can periodically check to see if the name becomes available by attempting to create a new repository with that name.
